src/storage/client_side_storage.ts

```diff
@@ -167,78 +167,87 @@ export function listModules(callback: ModulesCallback): void {
     if (cursor) {
       const value = cursor.value;
       const path = value.path;
+      const moduleType = _convertValueTypeToModuleType(value.type);
```

Comment:
Should this really be in the code since we really are only supporting Project?  (ie, there aren't legacy versions out in the wild)

Reply:
I removed it and added a note in the readme.